src/platform/icelake/include/platform/lib/memory.h

```diff
@@ -240,14 +240,31 @@
 #define HEAP_SYS_RT_X_COUNT512		8
 #define HEAP_SYS_RT_X_COUNT1024		4
 
+/* Heap section counts base */
+#define HEAP_COUNT64		128
+#define HEAP_COUNT128		128
+#define HEAP_COUNT256		96
+#define HEAP_COUNT512		8
+#define HEAP_COUNT1024		4
+#define HEAP_COUNT2048		2
+#define HEAP_COUNT4096		1
+
+#if HP_SRAM_SIZE < 0x200000
+#define RT_TIMES	1
+#define RT_SHARED_TIMES	1
+#else
+#define RT_TIMES	8
+#define RT_SHARED_TIMES	16
+#endif
+
 /* Heap section sizes for module pool */
-#define HEAP_RT_COUNT64			128
-#define HEAP_RT_COUNT128		64
-#define HEAP_RT_COUNT256		128
-#define HEAP_RT_COUNT512		8
-#define HEAP_RT_COUNT1024		4
-#define HEAP_RT_COUNT2048		1
-#define HEAP_RT_COUNT4096		1
```

Comment:
I want to address these large block sizes later. I dont think we have any users ?

Reply:
yes the 2KB and 4KB blocks have no users yet per my observation, maybe we can leave it for 256Bs and 512Bs for advance fine tuning.